draft-ietf-elegy-rfc8989bis.md

```diff
@@ -220,6 +225,36 @@ requirements (e.g. logging in to a single session and then immediately logging
 out). In the event of abuse of process, the leadership would then have months to
 adjust policy in response before the NomCom cycle begins.
 
+## Disruptive candidates
+
+Note that the normalization of consistent remote participation allows for a
+single individual to mount an attack that previously required coordination. By
+registering under a number of different identities over a year, an individual
+can make each of those identities NomCom eligible and then serve under any one
+of them that is selected for the NomCom.  Once selected, an individual could
+seek to disrupt the process or prevent the timely conclusion of its work.
+
+This attack is much harder to detect or prevent than equivalent attacks were
+previously, as it does not require coordination among multiple attendees.
+While the attacker cannot be sure of fee waivers for some or all of the
+different identities, the lower cost for remote participation also makes this
+attack more feasible than it would have been under prior rules.
+
+However, the voting member recall procedure in {{Section 5.7 of RFC8713}} exists
+to allow removal and replacement of disruptive figures.
+
+## Additional remedies
+
+Additional changes to the process to further obstruct attacks against the
+NomCom are beyond the scope of this document. However, a challenge process
+against volunteers with a suspicious reported affiliation, or that might be
+aliases of a single volunteer, could trigger an investigation by the
+Secretariat.
```

Comment:
The secretariat is not the IETF police. They would only provide data to leadership, and someone in leadership would need to make a decision. (Please check this wherever you talk about the secretariat doing something.)